src/Tools.js

```diff
@@ -12,19 +12,20 @@ class Tools {
 	 * within a bidi isolation block.
 	 *
 	 * @param {jQuery|string} $content Content to isolate
-	 * @param {boolean} returnRawHtml Force the return value to
+	 * @param {boolean} [returnRawHtml] Force the return value to
 	 *  be raw html. If set to false, will return the encompassed
 	 *  jQuery object.
 	 * @return {jQuery|string} BiDi isolated jQuery object or HTML
 	 */
-	static bidiIsolate( $content, returnRawHtml ) {
+	static bidiIsolate( $content, returnRawHtml = false ) {
 		const $result = $( '<bdi>' );
+
 		if ( typeof $content === 'string' ) {
-			$result.text( $content );
-		} else {
-			$result.append( $content );
```

Comment:
You can also switch to doing
```
if ( typeof $content === 'string' ) {
   $result.html( $.parseHTML( $content ) );
} else {
   $result.append( $content );
}
```
Since `.html()` can accept a parsed html string and doesn't require a jquery object.